python/lsst/ts/m2gui/controltab/tab_actuator_control.py

```diff
@@ -33,7 +57,516 @@ class TabActuatorControl(TabDefault):
+        selected_unit = ActuatorDisplacementUnit(
+            self._displacement_unit_selection.currentIndex() + 1
+        )
+
+        if selected_unit == ActuatorDisplacementUnit.Millimeter:
+            self._target_displacement.setDecimals(
+                self.model.utility_monitor.NUM_DIGIT_AFTER_DECIMAL_DISPLACEMENT
+            )
+        else:
+            self._target_displacement.setDecimals(0)
+
+    def _create_buttons_actuator_selection(self):
+        """Create the buttons of actuator selection.
+
+        Returns
+        -------
+        buttons : `list [PySide2.QtWidgets.QPushButton]`
+            Buttons of actuators.
+        """
+
+        buttons = list()
+
+        actuators = self.model.get_actuator_default_status(False)
+        for name in actuators.keys():
+            button = set_button(name, None, is_checkable=True, is_adjust_size=True)
+            buttons.append(button)
+
+        return buttons
+
+    @Slot()
+    def _callback_select_ring(self):
+        """Callback of the select-ring button in actuator selector. All
+        actuators in the specific ring will be selected."""
+
+        # Index begins from 0 instead of 1 in QComboBox
+        ring = Ring(self._ring_selection.currentIndex() + 1)
+
+        for actuator in self._buttons_actuator_selection:
+            if actuator.text().startswith(ring.name):
+                actuator.setChecked(True)
+
+    @Slot()
+    def _callback_clear_selection(self):
+        """Callback of the clear-selection button in actuator selector. The
+        selected actuators will be cleared/unchecked."""
+
+        for button in self._buttons_actuator_selection:
+            button.setChecked(False)
+
+    @Slot()
+    def _callback_actuator_start(self):
+        """Callback of the start button in actuator control. The cell
+        controller will start to move the actuators.
+
+        Returns
+        -------
+        actuators : `list`
+            Selected actuators to do the movement.
+        target_displacement : `float` or `int`
+            Target displacement of the actuators.
+        displacement_unit : enum `ActuatorDisplacementUnit`
+            Displacement unit.
+        """
+
+        actuators = list()
+        for idx, actuator_selection in enumerate(self._buttons_actuator_selection):
+            if actuator_selection.isChecked():
+                actuators.append(idx)
+
+        target_displacement = self._target_displacement.value()
+
+        # Index begins from 0 instead of 1 in QComboBox
+        displacement_unit = ActuatorDisplacementUnit(
+            self._displacement_unit_selection.currentIndex() + 1
+        )
+
+        run_command(
+            self.model.command_actuator,
+            CommandActuator.Start,
+            actuators,
+            target_displacement,
+            displacement_unit,
+        )
+
+        return actuators, target_displacement, displacement_unit
+
+    @Slot()
```

Comment:
This is a bit misleading, as Slot shall contain parameters for method. You fixed need to pass additional parameter by using partial to create function which doesn't need it, and then calling it. Technically this means this is normal function, not a Slot - certainly it cannot be connected to signal which doesn't pass any parameters. Better to drop @Slot() decorator.